<?xml version="1.0"?>
<launch>
    <!--node pkg="cloud_practice" type="cloud_loader" name="cloud_loader" />
    <node pkg="cloud_practice" type="cloud_filter" name="cloud_filter" />
    <node pkg="cloud_practice" type="cloud_downsampler" name="cloud_downsampler" /-->
    <arg name="input_pc_topic" default="/photoneo_center/sensor/points"/>
    <arg name="distance_threshold" default="0.001"/>
    <arg name="input_img_topic" default="/photoneo_center/sensor/image_color"/>
    <arg name="timespan" default="4"/>
    <arg name="service_name" default="first_input"/>
    
    <node pkg="cloud_practice" type="cloud_planar_segmenter_ishiyama2" name="cloud_planar_segmenter_ishiyama" output="screen" >
        <param name="input_pc_topic" value="$(arg input_pc_topic)"/>
        <param name="input_img_topic" value="$(arg input_img_topic)"/>
        <param name="distance_threshold" value="$(arg distance_threshold)"/>
        <param name="timespan" value="$(arg distance_threshold)"/>
        <param name="service_name" value="$(arg service_name)"/>
    </node>
    <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find cloud_practice)/rviz/planar.rviz"/-->
</launch>